libs/partners/anthropic/langchain_anthropic/chat_models.py

```diff
@@ -677,6 +688,29 @@ def _tools_in_params(params: dict) -> bool:
     )
 
 
+class _AnthropicToolUse(TypedDict):
+    type: Literal["tool_use"]
+    name: str
+    input: dict
+    id: str
+
+
+def _lc_tool_calls_to_anthropic_tool_use_blocks(
+    tool_calls: List[ToolCall],
+) -> List[_AnthropicToolUse]:
+    blocks = []
+    for tool_call in tool_calls:
```

Comment:
Nuno had a comment on this: https://github.com/langchain-ai/langchain/pull/20519/files#r1567919816
> can we support putting string content, if not empty, as an additional part of type text?